query-engine/schema-builder/Cargo.toml

```diff
@@ -9,6 +9,9 @@ schema = { path = "../schema" }
 prisma-models = { path = "../prisma-models" }
 once_cell = "1.3"
 
+[dev-dependencies]
+codspeed-criterion-compat = "1.1.0"
+
 [dev-dependencies.criterion]
 version = "0.4.0"
 default-features = false
```

Comment:
I guess this can now be removed since it's not directly used anymore (also a line below that github doesn't let me to comment on)

Reply:
No idea honestly, you tell me.

Reply:
`codspeed-criterion-compat` brings its own copy of criterion that it uses, so let's remove ours